tests/integrationtest/lib.go

```diff
@@ -2125,6 +2125,10 @@ func SetupIngressClass(t *testing.T, ingclassName, controller, infraSetting stri
 			t.Fatalf("error in adding IngressClass: %v", err)
 		}
 	}
+	if len(wait) > 0 {
```

Comment:
waiting for informer to reflect the change is better than waiting for constant time.
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1541/files#diff-65203e7df4245516a3ff56807dac4609eca286c0706102103b62a025a137e40cR2129

Reply:
updated